x-pack/test/ingest_manager_api_integration/apis/epm/install_by_upload.ts

```diff
@@ -9,47 +9,169 @@ import path from 'path';
 import expect from '@kbn/expect';
 
 import { FtrProviderContext } from '../../../api_integration/ftr_provider_context';
-import { warnAndSkipTest } from '../../helpers';
+import { skipIfNoDockerRegistry } from '../../helpers';
 
-export default function ({ getService }: FtrProviderContext) {
+export default function (providerContext: FtrProviderContext) {
+  const { getService } = providerContext;
   const supertest = getService('supertest');
   const dockerServers = getService('dockerServers');
-  const log = getService('log');
 
   const testPkgArchiveTgz = path.join(
```

Comment:
I imagine this is already done if the tests pass, but do these new fixtures have the new package changes? (I recall having to adjust `apache_0.1.4.tar.gz` during my PR)

Reply:
I need to update all package fixtures. Right now I adjusted the tests to the fact that the datasets (and assets related to them) are not found in the updated packages.

Reply:
I wonder if it is worth track unpacked version of the fixtures. binary diffs are not super useful 😅

Reply:
The downside of unpackaged packages is that they may contain a lot of files, which clutter PRs -- I accidentally opened one with so many files that the github review interface became unusable. I consider these fixtures like the packages contained in the dockerized registry -- unfortunately a bit difficult to handle, but they shouldn't change that often.

Also, I think I can reduce the number of these packages and move some of the validation tests to unit tests.